lib/babel-config.js

```diff
@@ -14,9 +12,9 @@ function validate(conf) {
 	const isValidShortcut = conf === 'default' || conf === 'inherit';
 
 	if (!conf || (typeof conf === 'string' && !isValidShortcut)) {
-		let message = colors.error(figures.cross);
+		let message = figures.cross;
 		message += ' Unexpected Babel configuration for AVA. ';
-		message += 'See ' + chalk.underline('https://github.com/avajs/ava#es2015-support') + ' for allowed values.';
+		message += 'See https://github.com/avajs/ava#es2015-support for allowed values.';
```

Comment:
Why this change? (I must admit I didn't read all of the back and forth earlier though… forgive me if I overlooked the answer)